types/slonik/index.d.ts

```diff
@@ -425,12 +392,10 @@ export interface InterceptorType {
 export function createInterceptorPreset(): InterceptorType[];
 
 export function createFieldNameTransformationInterceptor(configuration: {
-    format: string,
-    test?: (field: FieldType) => boolean
-}): InterceptorType;
-export function createQueryNormalizationInterceptor(configuration?: {
-    stripComments?: boolean;
+    format: string;
+    test?: (field: FieldType) => boolean;
 }): InterceptorType;
+export function createQueryNormalizationInterceptor(configuration?: { stripComments?: boolean }): InterceptorType;
```

Comment:
Did you mean to change this?

Reply:
This is only a formatting change from running prettier (something my editor does automatically (and will cause issues with any PR in the future where the contributor's editor also does this seeing as it's configured), so I think it's worth making the change.